projects/storefrontlib/src/lib/cms-lib/navigation/navigation.service.ts

```diff
@@ -1,10 +1,18 @@
-import { Injectable } from '@angular/core';
-import { CmsService } from '@spartacus/core';
+import { Injectable, Optional } from '@angular/core';
```

Comment:
Should I rename this class/file to navigation.component.service ?

Reply:
it is more readable imo. let's do that